src/Native/System.Net.Security.Native/pal_gssapi.cpp

```diff
@@ -102,9 +102,11 @@ extern "C" uint32_t NetSecurityNative_DisplayStatus(uint32_t* minorStatus,
     assert(outBuffer != nullptr);
 
     int statusType = isGssMechCode ? GSS_C_MECH_CODE : GSS_C_GSS_CODE;
+    uint32_t messageContent;
     GssBuffer gssBuffer {.length = 0, .value = nullptr};
     uint32_t majorStatus =
-        gss_display_status(minorStatus, statusValue, statusType, GSS_C_NO_OID, nullptr, &gssBuffer);
+        gss_display_status(minorStatus, statusValue, statusType, GSS_C_NO_OID, &messageContent, &gssBuffer);
+    (void)messageContent; // unused
```

Comment:
> The name of the parameter in the header file is message_content. So I used the same name with our naming conventions.

Interesting.  That's not what I see in all of the man pages online, nor in the headers on my Ubuntu install:
```
OM_uint32 *,        /* message_context */
```